_posts/2020-05-05-git-it-p1.md

```diff
@@ -2,6 +2,12 @@
 layout: post
 author: "Aashish Kumar"
 author_url: https://github.com/aashish-ak
+author_bio: "Description here"
```

Comment:
Don't you think adding this repeatedly in each post is very redundant, as many authors will be posting many blogs, author profile should be used instead of manually adding links in the YAML metadata. This is very important and is already being used in the KamandPrompt Blog minimal mistakes template [here](https://github.com/KamandPrompt/blog/blob/master/_data/authors.yml). You can see their [posts](https://raw.githubusercontent.com/KamandPrompt/blog/master/_posts/2019-01-10-welcome.md) only contain the author name, we can use the GitHub username instead as multiple authors can have the same name. We can discuss this on Slack if you want more insight.